.github/workflows/rolling-compatibility-iron-binary-build.yml

```diff
@@ -0,0 +1,22 @@
+name: Check Rolling Compatibility on Iron with Stack Build
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every morning to detect flakiness and broken dependencies
+    - cron: '03 1 * * *'
+
+jobs:
+  stack-build-on-iron:
+    uses: ./.github/workflows/reusable-industrial-ci-with-cache.yml
+    strategy:
+      fail-fast: false
+      matrix:
+        ROS_DISTRO: [iron]
+        ROS_REPO: [main, testing]
+    with:
+      ros_distro: ${{ matrix.ROS_DISTRO }}
+      ros_repo: ${{ matrix.ROS_REPO }}
+      target_workspace: ros_controls.rolling-on-iron.repos
+      ref_for_scheduled_build: master
+      rosdep_skip_keys: gazebo_ros2_control
```

Comment:
```suggestion
```

Reply:
it was not present any more in the humble workflow, so I thought it might not be necessary any more. let me run it once again

Reply:
Thank you!

Reply:
I think it should be fine, it's just that it is better to crosscheck just in case. Thanks man

Reply:
It works. you can revert it and It's it good to go! :)
Thanks